torch/nn/modules/module.py

```diff
@@ -281,22 +281,12 @@ def register_module_full_backward_pre_hook(
         This adds global state to the `nn.module` module
         and it is only intended for debugging/profiling purposes.
 
-    The hook will be called every time the gradients for the module are computed.
-    The hook should have the following signature::
+    Hooks registered using this function function in the same way as those
```

Comment:
function function is grammatically correct but hard to read. 'behaves" might be better

Reply:
Good point haha